### PR TITLE
Clarify Suspicious AWS Events

### PR DESCRIPTION
**Pull request by commit-dkp**
Rather than define what is unusual, define what is expected.

### CODE REVIEW

File: content/docs/ops/maintenance-list.md [hunk 1]
```diff
@@ -5,48 +5,73 @@ menu:
 title: Ongoing platform maintenance
 ---
 
-The platform requires regular support and maintenance activities to remain in a compliant state. If you are on support and you can’t complete any of the items personally, you are responsible for ensuring that an appropriate person does it.
+The platform requires regular support and maintenance activities to remain in a 
+compliant state. If you are on support and you can’t complete any of the items 
+personally, you are responsible for ensuring that an appropriate person does it.
```
Comment (rogeruiz) [inline content/docs/ops/maintenance-list.md:10]: Can we mention here that people should reach out at the end of program-level standups in order to get visibility with people who are in the best position to help?

File: content/docs/ops/maintenance-list.md [hunk 2]
```diff
@@ -5,48 +5,73 @@ menu:
...
 
 - Get the latest stemcell version from http://bosh.cloudfoundry.org/stemcells/.
 
-- Check https://ci.fr.cloud.gov/teams/main/pipelines/aws-light-stemcell-builder to ensure it has built the same version.
+- Check https://ci.fr.cloud.gov/teams/main/pipelines/aws-light-stemcell-builder 
+to ensure it has built the same version.
 
-- On each bosh director run `bosh deployments` and verify the stemcell in-use for each deployment is current.
+- On each bosh director run `bosh deployments` and verify the stemcell in-use 
+for each deployment is current.
```
Comment (rogeruiz) [inline content/docs/ops/maintenance-list.md:43]: We're talking about having a pipeline task for this. But it shouldn't stop this update. :+1:

for your eyes, https://gsa-tts.slack.com/archives/C0ENP71UG/p1496895499351167

File: content/docs/ops/maintenance-list.md [hunk 3]
```diff
@@ -61,7 +86,9 @@ Use the [AWS Console](http://docs.aws.amazon.com/govcloud-us/latest/UserGuide/go
 - RevokeSecurityGroupIngress
 - UpdateTrail
 
-If any suspicious activity is discovered discuss with appropriate team members and if necessary follow the [Security Incident Response Guide]({{< relref "docs/ops/security-ir.md" >}}).
+If you observe any non-Terraform activity, discuss the event(s) with the 
+indicated cloud.gov operator(s) and if necessary follow the
```
Comment (rogeruiz) [inline content/docs/ops/maintenance-list.md]: Can we make `indicated cloud.gov operator(s)` a link to [this Google doc](https://docs.google.com/spreadsheets/d/1mW3tphZ98ExmMxLHPogSpTq8DzYr5Oh8_SHnOTvjRWM/edit#gid=0)?

**Review by rogeruiz — CHANGES_REQUESTED** [review]
This looks great. 👍 here's some changes. Also in the future I think it's good to do whitespace changes to the docs, but it made it difficult to review. I'd say either focus on just the changes and whitespace / line break changes for the lines your editing or parts your editing **or** say in the opening comment that it should be viewed in _Rich Diff Mode_.

File: content/docs/ops/maintenance-list.md (hunk 1 above)
Comment (commit-dkp) [reply content/docs/ops/maintenance-list.md:10]: Added in https://github.com/18F/cg-site/pull/922/commits/96f57d379f819e0af0564b483d9e5d0c7b18600e .

**Review by rogeruiz — APPROVED** [review]
:shipit:

**Pull request merged**